docs/version0.27.md

```diff
@@ -33,6 +33,7 @@ The following new features and notable changes since v 0.26.0 are included in th
 - [Stop parsing the JAVA_OPTIONS environment variable](#stop-parsing-the-java_options-environment-variable)
 - [Global lock reservation enabled by default](#global-lock-reservation-enabled-by-default)
 - [Increase in default operating system stack size on PPC64 platforms](#increase-in-default-operating-system-stack-size-on-ppc64-platforms)
+- [New `-x` option added to `jpackcore` / `jextract`](##new-x-option-recognized-by-jpackcore-jextract)
```

Comment:
The link is not working in the draft (https://eclipse-openj9.github.io/openj9-docs-staging/808/version0.27) - one of the # symbols needs removing.

Reply:
Fixed.